marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/graph/graphSearchFacets.sjs

```diff
@@ -33,13 +33,24 @@ let assertions = [
   test.assertEqual(expectedNodeCount, resultsTest.total),
   test.assertEqual(expectedNodeCount, resultsTest.nodes.length),
   test.assertEqual(expectedEdgeCount, resultsTest.edges.length),
```

Comment:
I believe the comma at the end here will cause issues in the ML 9 nightly runs.

Reply:
That's correct! I just fixed that line